fofix/core/GameEngine.py

```diff
@@ -358,18 +358,24 @@ def __init__(self, config = None):
             log.warn("No stages\ folder found, forcing None setting for Animated Stage.")
             self.config.set("game","animated_stage_folder", "None") #MFH: force "None" when Stages folder can't be found
 
-
-
         try:
+            self.theme = None
             fp, pathname, description = imp.find_module("CustomTheme",[themepath])
-            theme = imp.load_module("CustomTheme", fp, pathname, description)
-            self.theme = theme.CustomTheme(themepath, themename)
+            try:
```

Comment:
Adding a `try` / `except` block is not useful here, I think… Finding & loading a module will raise the same exception for the same thing.

Reply:
No, find_module raises if it can't find the module (i.e. it doesn't exist) while load_module raises if it can't load the module (i.e. compile errors). The first is acceptable; a theme doesn't need to have a CustomTheme.py, so there's no particular need to log that situation. The second situation, however, is an error that the developer needs to know about.

Let me know if you still want some logging here.

Reply:
Ok. Then, log the error, please.